lib/global_enums.dart

```diff
@@ -0,0 +1,7 @@
+enum ShareFileType {
+  PDF, //ONLY PDF IS USED FOR NOW
```

Comment:
why do we need the rest if only PDF is being used? would everything work if we use docx? do we have tests for this?

Reply:
TBH, The enum was created automatically and because it doesn't affect anything to have more options there(non of which are implemented) I kept them there.